skorch/tests/test_lr_scheduler.py

```diff
@@ -0,0 +1,96 @@
+from torch.optim.lr_scheduler import *
+import torch.nn.functional as F
+import torch.nn as nn
+from torch.optim import SGD
+import torch
+
+from skorch.lr_scheduler import WarmRestartLR, LRScheduler
+from sklearn.datasets import make_classification
+from skorch.net import NeuralNetClassifier
+import numpy as np
+import pytest
+
+#TODO: Copying from pytorch test/test_optim. Should be working
+class SchedulerTestNet(torch.nn.Module):
+    def __init__(self, num_units=10, nonlin=F.relu):
+        super(SchedulerTestNet, self).__init__()
+
+        self.dense0 = nn.Linear(20, num_units)
+        self.nonlin = nonlin
+        self.dropout = nn.Dropout(0.5)
+        self.dense1 = nn.Linear(num_units, 10)
+        self.output = nn.Linear(10, 2)
+
+    def forward(self, X):
+        X = self.nonlin(self.dense0(X))
+        X = self.dropout(X)
+        X = self.nonlin(self.dense1(X))
+        X = F.softmax(self.output(X), dim=-1)
+        return X
```

Comment:
This whole class can be removed. You can simply use `classifier_module` as a fixture in your tests which is equivalent to `SchedulerTestNet`. For example `def test_lr_callback_init_policies(self):` would change to ` def test_lr_callback_init_policies(self, classifier_module):` and the invocation of `SchedulerTestNet` in subsequent tests would be substituted with `classifier_module`. This way you don't need to copy code.

Reply:
Done.